### PR TITLE
Fix tf 1.14.0 problems

### PR DESCRIPTION
**Pull request by justin1121**

### CODE REVIEW

File: tf_encrypted/convert/convert_test.py [hunk 1]
```diff
@@ -327,11 +327,12 @@ def test_keras_dense_convert(self):
         protocol='Pond',
     )
 
-  def test_keras_batchnorm_convert(self):
-    test_input = np.ones([1, 28, 28, 1])
-    self._test_with_ndarray_input_fn('keras_batchnorm',
-                                     test_input,
-                                     protocol='Pond')
+  # TODO(justin1121): how to re-enable???
```
[the remark this answers is not in the document]
Comment (justin1121) [reply tf_encrypted/convert/convert_test.py]: Its a bug in 1.14.0. We can open a bug anyway to turn this test back on with 1.15.0 or 2.0.

[the remark this answers is not in the document]
Comment (justin1121) [reply tf_encrypted/convert/convert_test.py]: I'll update the comment too.

File: setup.py [hunk 2]
```diff
@@ -14,12 +14,12 @@
     ]},
     python_requires=">=3.5",
     install_requires=[
-        "tensorflow >=1.12.0, <2",
+        "tensorflow >=1.13.1, <2",
```
[the remark this answers is not in the document]
Comment (justin1121) [reply setup.py]: You mean removing tensorflow from install_requires or just leaving 1.12.0?

[the remark this answers is not in the document]
Comment (jvmancuso) [reply setup.py]: leaving it as 1.12.0

[the remark this answers is not in the document]
Comment (justin1121) [reply setup.py]: Sounds good.

**Pull request merged**